packages/react-scripts/scripts/build.js

```diff
@@ -157,9 +170,14 @@ function build(previousSizeMap) {
         // "homepage": "http://mywebsite.com"
         console.log('You can control this with the ' + chalk.green('homepage') + ' field in your '  + chalk.cyan('package.json') + '.');
         console.log();
+        console.log('Alternatively, you may change the absolute URL for all files by setting a CDN_URL environment variable');
+        console.log();
       } else {
         // no homepage
         console.log('To override this, specify the ' + chalk.green('homepage') + ' in your '  + chalk.cyan('package.json') + '.');
+        console.log();
+        console.log('Alternatively, you may change the absolute URL for all files by setting a CDN_URL environment variable');
```

Comment:
I think these logs are too noisy. Let’s remove them as most users won’t need them. Those who need them will look for “CDN” in “Usage Guide”.